src/Symfony/Component/AssetMapper/Command/ImportMapRequireCommand.php

```diff
@@ -110,28 +127,45 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             );
         }
 
-        $newPackages = $this->importMapManager->require($packages);
+        if ($input->getOption('dry-run')) {
+            $newPackages = $this->importMapManager->requirePackages($packages, new ImportMapEntries());
+        } else {
+            $newPackages = $this->importMapManager->require($packages);
+        }
 
         $this->renderVersionProblems($this->importMapVersionChecker, $output);
 
-        if (1 === \count($newPackages)) {
-            $newPackage = $newPackages[0];
-            $message = \sprintf('Package "%s" added to importmap.php', $newPackage->importName);
+        $newPackageNames = array_map(fn (ImportMapEntry $package): string => $package->importName, $newPackages);
 
-            $message .= '.';
+        if (1 === \count($newPackages)) {
+            $messages = [\sprintf('Package "%s" added to importmap.php.', $newPackageNames[0])];
         } else {
-            $names = array_map(fn (ImportMapEntry $package) => $package->importName, $newPackages);
-            $message = \sprintf('%d new items (%s) added to the importmap.php!', \count($newPackages), implode(', ', $names));
+            $messages = [\sprintf('%d new items (%s) added to the importmap.php!', \count($newPackages), implode(', ', $newPackageNames))];
         }
 
-        $messages = [$message];
+        if ($io->isVerbose()) {
+            $io->table(
+                ['Package', 'Version', 'Path'],
+                array_map(fn (ImportMapEntry $package): array => [
+                    $package->importName,
+                    $package->version ?? '-',
+                    // BC layer for FrameworkBundle < 7.3
```

Comment:
this should not mention `FrameworkBundle < 7.3` IMO. This is a BC layer for the case where the project dir is not provided, which may also impact other packages than FrameworkBundle. It is related to the BC layer of the current class.

Reply:
Done, thanks !

Reply:
@stof Is their anything else here ?